bgpd/bgp_filter.h

```diff
@@ -25,6 +25,13 @@ struct as_filter {
 	int64_t seq;
 };
 
+
+struct bp_as_excl_list {
```

Comment:
What does it mean `bp`? Back-pointer? Then useless meaning of this IMO, just better use a readable naming instead.

Reply:
And please name the structure without shortening it, it just lowers the readability... (we already have non-shortened version `exclude_aspath_acl`)